modules/ROOT/pages/auradb/managing-databases/migration-readiness.adoc

```diff
@@ -103,14 +105,115 @@ Like the Cypher deprecations table, the last column links to a view of this spec
+We highly recommend running a test migration before attempting the live migration.
+
+[NOTE]
+====
+During the migration, the migration target instance might be shown with a few different status on the instance page like LOADING or OVERWRITING.
+Do not access the instance before the migration is safely finished. The progress of migration can be seen in the Migration Readiness Report of the original instance.
+====
+
+=== Run a test migration
+
+Use the "Run test migration" buttons at the top or bottom of the page and then follow the steps outlined in the dialog boxes.
+
+The steps of running a test migration are:
+
+* Carefully read and act upon the steps described in the "Read before test migration" dialog.
+Proceed only if you made the appropriate preparations (e.g. backups of your configurations).
+* Configure a target instance, as described in the next section.
+** If you have selected a new instance to migrate to: Download the new credentials for that instance.
+* Wait for the migration to finish.
+* Follow all steps outlined in "Next steps before finalizing the test migration" at the top of the Migration Readiness Report page.
+This includes all your testing on the migrated instance.
+* Once you are done with testing, click the "Finalize test migration" button and complete the dialog to remove your test instance.
+
+You can repeat test migrations or run them in parallel as much as need.
+Please be aware that running those instances will incur the same cost as running any other instance of that size.
+
+==== Configure target instance
+
+An instance can either be migrated to a brand-new instance or an instance that is already running the latest version of Aura and that fits the memory and storage configuration of the original instance.
+This means that if you choose the second option the instance you want to migrate to has to have at least the same amount of memory and storage as the original one.
+
+Note that cloning into an existing instance will overwrite all of its existing data and name.
+This action cannot be undone and may take longer than cloning to a new instance.
+It is advised to take a snapshot and download it before continuing if you still have data on that instance you want to keep.
+
+[NOTE]
+====
+In the process of migrating to a test instance, that instance, regardless if new or existing, will get a new name.
+It will start with "[Testing]", followed by (most of) the original instances name and a test counter in parentheses e.g. "[Testing] original name (1)".
+====
+
+==== Testing the migrated instance
+
+Once you see the following box on the Migration Readiness Report, your migrated instance is ready for testing.
+Follow the steps described and test your instance to make sure your live migration will go smoothly.
+
+image::mrr-test-instance-ready.png[]
+
+==== Finalize test migration
+
+Once you are done with testing, click the "Finalize test migration" button and the following dialog will be displayed.
```

Comment:
```suggestion
Once you are done with testing, use the "Finalize test migration" button.
You will be asked to acknowledge the finalization since the test instance is deleted in the process.
You can skip this step and keep the test instance, but this incurs a cost.
Therefore, to minimize costs if you test manually, don't forget to delete the test instance when you are done.
```